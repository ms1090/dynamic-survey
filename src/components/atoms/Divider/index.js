import React from 'react';

/**
 * Display a horizontal divider line with 20px top & bottom margin
 * @return {React.Element} Return Divider component UI
 */
const Divider = () => (
  <hr style={{ margin: '20px 0' }} />
);

export default Divider;
