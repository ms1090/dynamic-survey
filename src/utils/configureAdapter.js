import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
// configure adapter to use enzyme in unit test cases
configure({ adapter: new Adapter() });
