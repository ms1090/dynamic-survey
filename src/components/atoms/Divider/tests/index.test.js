import React from 'react';
import { shallow } from 'enzyme';
import Divider from '..';
import toJson from 'enzyme-to-json';
import 'utils/configureAdapter';

describe('Divider Component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<Divider />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
