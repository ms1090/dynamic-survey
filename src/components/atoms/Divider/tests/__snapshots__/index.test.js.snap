// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Component should render correctly 1`] = `
<hr
  style={
    Object {
      "margin": "20px 0",
    }
  }
/>
`;
